packages/wget2.rb

```diff
@@ -41,8 +48,27 @@ class Wget2 < Autotools
                   --with-bzip2 \
                   --with-lzma'
 
-  def self.install
-    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+  def self.patch
+    File.write 'gettext.patch', <<~GETTEXT_PATCH_EOF
+      diff -Npaur a/configure.ac b/configure.ac
+      --- a/configure.ac	2025-11-24 16:32:01.716022174 -0500
+      +++ b/configure.ac	2025-11-24 16:33:02.520730059 -0500
+      @@ -417,10 +417,6 @@ m4_ifdef([AM_GNU_GETTEXT], [
+       ], [
+         have_po=no
+       ])
+      -m4_ifdef([AM_GNU_GETTEXT_VERSION], [
+      -#do not indent here
+      -AM_GNU_GETTEXT_VERSION([0.21])
+      -])
+       AM_CONDITIONAL([HAVE_PO], [ test "$have_po" = "yes" ])
+    GETTEXT_PATCH_EOF
+    system 'patch -Np1 -i gettext.patch'
```

Comment:
What is the purpose of this patch? Should we send it upstream in some form?

Reply:
Fair enough. Can you put `It's gettext related somehow. The run of autopoint during the autoreconf -fiv invocation cable.` in a comment above the patch so I can remember its purpose if i get around to trying to fix it?